robolectric/src/main/java/org/robolectric/util/ActivityController.java

```diff
@@ -241,4 +242,71 @@ public void run() {
         .resume()
         .visible();
   }
+  
+  /**
+   * Performs a configuration change on the Activity.
+   *  
+   * @param newConfiguration The new configuration to be set.
+   * @return May return a new ActivityController for a new Activity instance if the configuration change
+   * 		is not handled by the activity itself, otherwise, itself.
+   */
+  public ActivityController<T> configurationChange(final Configuration newConfiguration) {
+    final Configuration currentConfig = component.getResources().getConfiguration();
+    final int changedBits = currentConfig.diff(newConfiguration);
+    currentConfig.setTo(newConfiguration);
+    
+    // Can the activity handle itself ALL configuration changes?
+    if ((getActivityInfo(component.getApplication()).configChanges & changedBits) == changedBits) {
+      shadowMainLooper.runPaused(new Runnable() {
+        @Override
+        public void run() {
+          ReflectionHelpers.callInstanceMethod(Activity.class, component, "onConfigurationChanged",
+            ClassParameter.from(Configuration.class, newConfiguration));
+        }
+      });
+
+      return this;
+    } else {
+    	@SuppressWarnings("unchecked")
+		final ActivityController<T> recreatedActivityController = Robolectric.buildActivity(
```

Comment:
ok, I'll make this method return `T`, and document it. If the user would want to further manage the new activity however, he would need to manually create a new `ActivityController.of(Robolectrict.getShadowsAdapter(), newActivityInstance)`, since `component` is final and this controller itself will still point to the old, now destroyed activity.